app/models/manageiq/providers/embedded_ansible/automation_manager/playbook.rb

```diff
@@ -2,4 +2,46 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook <
   ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptPayload
 
   has_many :jobs, :class_name => 'OrchestrationStack', :foreign_key => :configuration_script_base_id
+
+  def run(options, userid = nil)
+    options[:playbook_id] = id
+    options[:userid]      = userid || 'system'
+    options[:name]        = "Playbook: #{name}"
+    miq_job = ManageIQ::Providers::EmbeddedAnsible::AutomationManager::PlaybookRunner.create_job(options)
```

Comment:
@bzwei 
Do we need try to make sure we have a miq_job?

Reply:
I don't think so. `create_job` is a straight forward method.